source/about/staff.rst

```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 |                                   |                                    |
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
```

Comment:
s/Ore Back-end Manager/ Ore Development Manager

Reply:
And mbaxter = Ore Community Manager, right @Zidane ?

Reply:
`Ore Development Manager`

Reply:
`Ore Development Manager`